### ISSUE TITLE
Resource type not found: Cron::Special

### ISSUE THREAD

**Issue by Helkar** [voxpupuli/puppet-cron#124]
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 7.26.0
- Ruby: 2.7.8
- Distribution: Ubuntu 18.04
- Module version: 4.1.0

## What are you seeing
An evaluation problem running puppet apply command

## What behaviour did you expect instead

No evaluation error

## Output log
Error: Evaluation Error: Resource type not found: Cron::Special (file: /etc/puppetlabs/code/modules/cron/manifests/job.pp, line: 30, column: 3) 
## Any additional information you'd like to impart

I have downgraded the module to 4.0.0 and the problem went away

**Comment by alexjfisher** [comment]
Odd.  Is this repeatable?  The `Cron::Special` type has been in the module for 6 years and I can't see anything related in the 4.1.0 changelog.

**Comment by Helkar** [comment]
Yes, every single time I executed the puppet apply it failed with the same reason. We are just defining some cron jobs, this is an example:

cron::job { '<cronjob name>':
    command     => '/usr/local/bin/<script>.sh',
    user        => 'root',
    hour        => '*',
    minute      => '0',
    environment => ['MAILTO=""'],
    require     => File['/usr/local/bin/<script>.sh'],
  }
